pkg/sentry/arch/syscalls_arm64.go

```diff
@@ -52,11 +52,12 @@ func (c *context64) SyscallArgs() SyscallArguments {
 // RestartSyscall implements Context.RestartSyscall.
 func (c *context64) RestartSyscall() {
 	c.Regs.Pc -= SyscallWidth
-	c.Regs.Regs[8] = uint64(restartSyscallNr)
+	c.Regs.Regs[0] = uint64(c.OrigR0)
 }
```

Comment:
This doesn't need to re-set R8 because it hasn't been clobbered since entry, right? A comment here would be helpful.

Reply:
Hi @prattmic 
We will backup R0 to OrigR0 when entering doSyscall() (pkg/sentry/kernel/task_syscall.go".
And this implementation will be delivered by another PR.
Please check the same backup method in Linux:
https://github.com/torvalds/linux/blob/master/arch/arm64/kernel/syscall.c#93